sql/core/src/test/resources/sql-tests/inputs/pgSQL/window.sql

```diff
@@ -0,0 +1,903 @@
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+-- Window Functions Testing
+-- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/window.sql
+
+CREATE TEMPORARY VIEW tenk2 AS SELECT * FROM tenk1;
+
+CREATE TABLE empsalary (
+    depname string,
+    empno integer,
+    salary int,
+    enroll_date date
+) USING parquet;
+
+INSERT INTO empsalary VALUES
+('develop', 10, 5200, '2007-08-01'),
+('sales', 1, 5000, '2006-10-01'),
+('personnel', 5, 3500, '2007-12-10'),
+('sales', 4, 4800, '2007-08-08'),
+('personnel', 2, 3900, '2006-12-23'),
+('develop', 7, 4200, '2008-01-01'),
+('develop', 9, 4500, '2008-01-01'),
+('sales', 3, 4800, '2007-08-01'),
+('develop', 8, 6000, '2006-10-01'),
+('develop', 11, 5200, '2007-08-15');
+
+SELECT depname, empno, salary, sum(salary) OVER (PARTITION BY depname) FROM empsalary ORDER BY depname, salary;
+
+SELECT depname, empno, salary, rank() OVER (PARTITION BY depname ORDER BY salary) FROM empsalary;
+
+SELECT four, ten, SUM(SUM(four)) OVER (PARTITION BY four), AVG(ten) FROM tenk1
```

Comment:
Could you keep the original PostgreSQL comment like the following, too?
```
-- with GROUP BY
```

Actually, those comments are markers. There are more missing comments in this file.

Reply:
Yes, I can add them. But I just checked and looks like the PgSQL file is way different from the time that I started to port it. What should I do? Migrate to the new version of the file or ignore the new additions and follow through the old one?

Reply:
We are porting `REL_12_BETA2` tag instead of `branch`. Tag is not changed.
- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/window.sql